spec/rails_helper.rb

```diff
@@ -46,7 +46,9 @@
   # Capybara::Selenium::Driver.new(app, browser: :chrome, desired_capabilities: capabilities)
 end
 
-# use selenium_chrome to actually see what's doing
+# To actually see the tests in the browser locally,
+# use selenium_chrome instead of selenium_chrome_headless
+# Capybara.javascript_driver = :selenium_chrome
```

Comment:
Just added a commented out example to make it easier to not have to remember the syntax when we want to run the feature tests in chrome locally.